securedrop/tests/test_store.py

```diff
@@ -27,6 +27,12 @@ def test_verify(self):
             store.verify(os.path.join(config.STORE_DIR, '..', 'etc', 'passwd'))
         with self.assertRaises(store.PathException):
             store.verify(config.STORE_DIR + "_backup")
+        STORE_DIR = config.STORE_DIR
+        with self.assertRaisesRegexp(store.PathException,
+                                     'is not absolute'):
```

Comment:
In `store.py` there are two exceptions that can be thrown that would match the regexp here - when the path passed as an arg is not absolute (line 44) _and_ when `config.STORE_DIR` is not absolute (line 36). Let's change the regexp to be more precise so that this assert can only pass if the behavior on line 36 is correct. For example:

```
with self.assertRaisesRegexp(store.PathException,
            'config.STORE_DIR\(\S*\) is not absolute'):
```